README.md

```diff
@@ -0,0 +1,56 @@
+XMR to BTC Atomic Swap
+======================
+
+This repository is a proof of concept for atomically swapping XMR for BTC.
```

Comment:
@luckysori,  what do you think: should we add a short section describing the protocol in a few words?

Reply:
A link to the blog post could work for the time being.

Reply:
I'd be for a standalone solution as I think the protocol might evolve and we are more likely to update the readme than the blogpost.

Reply:
In that case I think we should write a specification as a separate document and link to it in the readme. If we don't want to invest time into that yet, we could maybe copy the "Long story short" section from the blog post here. Maybe even with the diagram!

Reply:
👍 for copying "Long story short" + diagram.

Reply:
Done.